metricbeat/module/logstash/node_stats/data_test.go

```diff
@@ -21,9 +21,9 @@ package node_stats
 
 import (
 	"fmt"
-	"io/ioutil"
 	"net/http"
 	"net/http/httptest"
+	osutil "os"
```

Comment:
Why rename it to `osutil` instead of just using `os`?

Reply:
In the package, it define a os struct which is conflict to os package name.

https://github.com/elastic/beats/blob/543e1f26c9968452812acfb67444020fcdc03f1c/metricbeat/module/logstash/node_stats/data.go#L82-85

```type os struct {
	CPU    cpu    `json:"cpu"`
	CGroup cgroup `json:"cgroup,omitempty"`
}
```

I believe that renaming imported packages has a smaller impact compared to renaming an already used OS struct.

Reply:
I missed that, thanks for the clarification!